ner/src/main/java/edu/illinois/cs/cogcomp/ner/LbjTagger/LearningCurveMultiDataset.java

```diff
@@ -240,21 +239,20 @@ public static void getLearningCurve(Vector<Data> trainDataSet, Vector<Data> test
         }
     }
 
-    /*
+    /**
```

Comment:
If this info isn't already in the NER README, please add it there in a section relating to training -- and give some idea by what you mean by 'a lot of samples'